src/Kiota.Builder/Extensions/StringExtensions.cs

```diff
@@ -218,6 +218,7 @@ public static string CleanupSymbolName(this string? original)
         {'=', "Equal"},
         {'>', "GreaterThan"},
         {'?', "QuestionMark"},
+        {'~', "Tilda" },
```

Comment:
```suggestion
        {'~', "Tilde" },
```

I believe the official name of this character is Tilde
https://en.wikipedia.org/wiki/Tilde